library/src/morpheus-master/sass/modules/tool/assignments/_assignments.scss

```diff
@@ -412,77 +417,77 @@
 
 	.contentReviewIconWarn {
 		@extend .fa, .fa-exclamation-triangle;
-		color: #FFC107;
+		color: var(--warnBanner-color);
 	}
 
 	.contentReviewIconPending {
 		@extend .fa, .fa-hourglass-half;
-		color: #2196F3;
+		color: var(--infoBanner-color);
 	}
 
 	.contentReviewIconNoService {
 		@extend .fa, .fa-cog;
-		color: #212121;
+		color: var(--sakai-text-color-1);
 	}
 
 	.contentReviewIconThreshold-1 {
 		@extend .contentReviewIcon;
-		color: #F44336;
+		color: var(--errorBanner-color);
 	}
 
 	.contentReviewIconThreshold-1-draft {
 		@extend .contentReviewIconDraft;
-		color: #F44336;
+		color: var(--errorBanner-color);
 	}
 
 	.contentReviewIconThreshold-2 {
 		@extend .contentReviewIcon;
-		color: #FF9800;
+		color: var(--warnBanner-color);
 	}
 
 	.contentReviewIconThreshold-2-draft {
 		@extend .contentReviewIconDraft;
-		color: #FF9800;
+		color: var(--warnBanner-color);
 	}
 
 	.contentReviewIconThreshold-3 {
 		@extend .contentReviewIcon;
-		color: #FFD600;
+		color: var(--warnBanner-color);
 	}
 
 	.contentReviewIconThreshold-3-draft {
 		@extend .contentReviewIconDraft;
-		color: #FFD600;
+		color: var(--warnBanner-color);
 	}
```

Comment:
We might need more colours for these six levels to better illustrate the various threshold levels.